onnx/version_converter/convert.cc

```diff
@@ -73,13 +73,15 @@ void DefaultVersionConverter::convert_graph(
       debug(std::string("Finding schema for ") + std::string(cur_op->kind().toString()));
       const std::string op_name = cur_op->kind().toString();
       if (op_name == "ConstantFill") {
-        std::cerr
-            << "Warning: skipping schema search for experimental op 'ConstantFill' and keeping the op as is. "
-               "Please be advised the converted model may not be working properly if target runtime does not support this "
-               "experimental op."
-            << std::endl;
+        if (DEBUG)
```

Comment:
tiny nit: perhaps put cerr into brackets? It's up to you, but it will look clearer to me.

Reply:
Agreed, it's better to have braces here. The linter didn't remind me.